modmesh/core.py

```diff
@@ -105,6 +107,34 @@ def _load():
     for name in __all__:
         globals()[name] = getattr(_impl, name)
 
+    # Try to find the thirdparty, if failed to find it
+    # modmesh will raise ImportError and remind the user.
+    filename = os.path.join('thirdparty')
+    path = os.getcwd()
+    try:
+        while True:
```

Comment:
You need a unit test for the loop so that when it goes wrong and into infinite, there is a place to add regression.